CHANGELOG.md

```diff
@@ -47,7 +49,7 @@ Highlights of this release are a new way to specify formatter options closer to
 * Usage of `formatter_options` is deprecated, instead please use the new tuple way
 
 ### Features (User Facing)
-* benchee now uses the maximum precision available to measure which on Linux and OSX is nanoseonds instead of microseconds. Somewhat surprisingly `:timer.tc/1` always cut down to microseconds although better precision is available.
+* Benchee now uses the maximum precision available to measure which on Linux and OSX is nanoseonds instead of microseconds. Somewhat surprisingly `:timer.tc/1` always cut down to microseconds although better precision is available.
```

Comment:
actually, this was on purpose :sweat_smile: to me it is... `benchee` lower case. I thought elastic would handle it the same way, but they don't they also often name their stuff upper case :thinking: 

Probably no way around it. Lower case always seemed cooler to me.

Reply:
I think it's fine, it always felt kinda weird. I might change my mind again but for now it's fine.